include/dgl/aten/array_ops.h

```diff
@@ -36,6 +36,20 @@ inline bool IsNullArray(NDArray array) {
   return array->shape[0] == 0;
 }
 
+/*!
+ * \return Whether all the input arrays are null arrays.
+ */
+inline bool IsAllNullArray(const std::vector<NDArray>& arrays) {
+  bool all_null = true;
```

Comment:
how about utilizing `std::all_of()` from `<algorithm>` which is more straightforward?

Reply:
Good idea.